TODO.txt

```diff
@@ -19,13 +15,6 @@ PG2:
   - change win download repo to msys when they start uploading
   - reindex command - drops index and re-does everything
 
-* Videos
-  - add transcription to search
-  - add to related items
-
-* release notes box
```

Comment:
For a moment I thought that including the release notes could be interesting, but googling it's too easy to find them, so maybe it's not worthy: https://github.com/git/git/tree/master/Documentation/RelNotes